packages/blockfrost/test/e2e/queryTransactions.test.ts

```diff
@@ -164,7 +164,9 @@ describe('blockfrostWalletProvider', () => {
       expect(txs.length).toBeGreaterThanOrEqual(1);
       expect(txs[0].id).toBe('2822d491a890b40cd2a22003b81a97f63c2b8c373b1b0b8dfa1598739fe34c06');
     });
-    it('multiple address types', async () => {
+    // Failing because returned transactions are grouped by address
+    // TODO: update and reenable test when we decide behaviour
+    it.skip('multiple address types', async () => {
```

Comment:
On second thought @mkazlauskas, seeing as we have no CI for e2e tests I could leave this enabled failing.

Reply:
I think it's better to skip until the issue is resolved